### ISSUE TITLE
returnNew/returnOld options are not respected when updating vertices

### ISSUE THREAD

**Issue by uditha-atukorala** [arangodb/arangojs#677]
ArangoJS: `v6.14.1`
ArangoDB: `v3.6.4`

When `returnNew`/`returnOld` options passed to graph vertex collection `update()` method, `new`/`old` properties are not set in return object.

It looks like the issue is at (`new`/`old` properties are in `res.body`) https://github.com/arangodb/arangojs/blob/f310fb31597362ec7947c308530e1d60796d8d21/src/graph.ts#L155

**Comment by pluma** [comment]
Yes, this is a known (but undocumented) bug in v6 that has been fixed in v7.

See the last bullet point of https://github.com/arangodb/arangojs/blob/v7.0.0-rc-1/CHANGELOG.md#graph-api-2

**Comment by uditha-atukorala** [comment]
Are there any plans to fix this in v6? I tried moving to v7 but it's not straight forward so will probably have to stick to v6 for a while.
